#Exception handling
try:

    l=10
    w=0
    l/w
except Exception:
    print("Plz Enter the valid input")
except nameError:
    print("input has been used")
